src/main/java/com/github/angleshq/angles/listeners/cucumber/AnglesCucumber2Adapter.java

```diff
@@ -0,0 +1,102 @@
+package com.github.angleshq.angles.listeners.cucumber;
+
+import com.github.angleshq.angles.assertion.junit.AnglesJUnit5Assert;
+import com.github.angleshq.angles.basetest.AbstractAnglesTestCase;
+import cucumber.api.Result;
+import cucumber.api.TestStep;
+import cucumber.api.event.*;
+import cucumber.api.formatter.Formatter;
+import cucumber.runner.PickleTestStep;
+import gherkin.pickles.PickleCell;
+import gherkin.pickles.PickleRow;
+import gherkin.pickles.PickleTable;
+
+public class AnglesCucumber2Adapter extends AbstractAnglesTestCase implements Formatter {
+    //initialize the assertion instance
+    protected AnglesJUnit5Assert doAssert = new AnglesJUnit5Assert();
+
+    //event handlers - Angles loggers
+    protected EventHandler<TestStepStarted> handleTestStepFinished = new EventHandler<TestStepStarted>() {
+        @Override
+        public void receive(TestStepStarted event) {
+            TestStep testStep = event.testStep;
+            if (testStep.getClass() == PickleTestStep.class) {
+                String text = event.testStep.getStepText();
+                anglesReporter.startAction(text);
+                anglesReporter.info(text);
+                String argumentTable = "\n";
+                try {
+                    PickleTable argument = (PickleTable) testStep.getStepArgument().get(0);
+                    for (PickleRow row : argument.getRows()) {
+                        argumentTable += "\n| ";
+                        for (PickleCell cell : row.getCells()) {
+                            argumentTable += cell.getValue() + " | ";
+                        }
+                    }
+                    anglesReporter.info(argumentTable);
+                } catch (Exception e) {
+
```

Comment:
Maybe add an info here, that we weren't able to display the table?

Reply:
added